assets/src/edit-story/components/canvas/eyedropperLayer.js

```diff
@@ -22,6 +22,11 @@ import styled from 'styled-components';
 import { rgba, readableColor } from 'polished';
 import { FULLBLEED_RATIO } from '@web-stories-wp/units';
 
+/**
+ * WordPress dependencies
+ */
+import { Spinner } from '@wordpress/components';
```

Comment:
We don't import any UI components from `@wordpress/`, ever

Instead import `CircularProgress` from `../circularProgress` or `LoadingSpinner` from `@web-stories-wp/design-system`

Reply:
Oh, right, I saw those imports but they are in `web-stories-block` 🤦🏻